crates/ra_hir/src/nameres/tests.rs

```diff
@@ -364,6 +364,101 @@ fn module_resolution_works_for_raw_modules() {
     "###);
 }
 
+#[test]
+fn module_resolution_decl_path() {
```

Comment:
Lets move these and other tests that target specifically how `mod`s are resoved into a new `nameres/tests/mods.rs` submodule?

Reply:
Yes, of course